### ISSUE TITLE
Central translation repository

### ISSUE THREAD

**Issue by filips123** [sampotts/plyr#1261]
Currently, the only possible way to use translation for Plyr is to get `i18n` object from somewhere and add it to Plyr. This have several weaknesses because users (developers) would need to know location of file for each translation.

There should be some central translation "repository" (the best options is some folder in this repository, for example `src/locales`) which would have all translations in JSON files.
Users (developers) would than just need to specify language name to get translation.

**Comment by sampotts** [comment]
I guess the issue is including or loading the JSON somehow. Building localized bundles or getting users to include i18n files that override the config is another way. Obviously loading the remote JSON file is possible but slower unless cached in local storage or something.

**Comment by filips123** [comment]
@sampotts Yes, JSON files should be bundled directly in Plyr or downloaded from web and cached, as you said.

**Comment by filips123** [comment]
@sampotts

I will make that translations will be saved into `src/locales/{lang}.json` and bundled to Plyr file.

I have problem that this require `fs` module to check if file exists and `require` to load it. They are not available in browsers with gulp.
How can I use them there? Is it possible to build Plyr with webpack? How?

**Comment by filips123** [comment]
In the future, you should add support for user changing of language in UI options. I've already added language name field for this (in src/locales/index.js).

See #1270.

**Comment by HybridSolutions** [comment]
This player also uses a similar translation feature

https://github.com/mediaelement/mediaelement/blob/master/docs/installation.md#language
